manifests/base/runtimes/kustomization.yaml

```
@@ -3,3 +3,4 @@ kind: Kustomization
 resources:
   - torch_distributed.yaml
   - mpi_distributed.yaml
+  - deepspeed_distributed.yaml
```

```suggestion
  - deepspeed_distributed.yaml
  - mpi_distributed.yaml
  - torch_distributed.yaml
```

If there are no strict restriction on installation order, we'd better list them alphabetically:)